src/main/resources/org/sagebionetworks/web/client/widget/docker/DockerRepoWidgetViewImpl.ui.xml

```diff
@@ -27,6 +27,7 @@
 					</b:Panel>
 				</b:Column>
 				<b:Column size="XS_12, MD_6">
+					<bh:Div ui:field="dockerCommitListContainer" addStyleNames="highlight-box" />
```

Comment:
instead of highlight-box method, see FilesTabViewImpl.ui.xml (to support contextual help).  this also applies to the provenance container.